src/sentry/static/sentry/app/components/sampleEventBanner.jsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import Reflux from 'reflux';
+import styled from 'react-emotion';
+
+import Alert from 'app/components/alert';
+import ConfigStore from 'app/stores/configStore';
+import {t} from 'app/locale';
+
+const SampleEventBanner = createReactClass({
+  displayName: 'sampleEventBanner',
+  propTypes: {
+    organization: PropTypes.object,
+  },
+
+  mixins: [Reflux.listenTo(ConfigStore, 'onConfigStoreUpdate')],
+
+  getInitialState() {
+    return {
+      sentFirstEvent: false,
+    };
+  },
+  componentDidMount() {
+    this.sentFirstEvent();
+  },
+
+  onConfigStoreUpdate(config) {
+    if (config.sentFirstEvent) {
```

Comment:
we should check this only if `this.state.sentFirstEvent` is false, otherwise it will flip flop